src/collective/liveblog/static/styles.css

```diff
@@ -4,10 +4,6 @@
 }
 .portaltype-liveblog article .field input[type="text"] {
     width: 100%;
-    line-height: 125%;
-    color: black;
-    letter-spacing: -0.05em;
-    margin: inherit -0.05em;
```

Comment:
Jigsaw complained for ``margin: inherit -0.05em;`` and I removed some other nonsense stuff (why should we style the input text of the title.. for me looks fine like this)